pkg/config/config.go

```diff
@@ -130,6 +138,9 @@ type TestConfig struct {
 	// OperatorSkip is a comma-delimited list of operator names to ignore health checks from. ex. "insights,telemetry"
 	OperatorSkip string `env:"OPERATOR_SKIP" sect:"tests" default:"insights" yaml:"ginkgoFocus"`
 
+	// SkipClusterHealthChecks skips the cluster health checks. Useful when developing against a running cluster.
+	SkipClusterHealthChecks bool `env:"SKIP_CLUSTER_HEALTH_CHECKS" sect:"tests" yaml:"skipClusterHealthChecks"`
```

Comment:
I think it's better to specify `default:"false"` here

Reply:
Good catch -- had meant to do this.